src/cart/crt_hg.h

```diff
@@ -52,6 +52,14 @@ crt_na_type_is_ofi(int na_type)
 	       (na_type < CRT_NA_OFI_COUNT);
 }
 
+static inline bool
+crt_na_type_is_slow(int na_type)
+{
+	if (na_type == CRT_NA_OFI_SOCKETS || na_type == CRT_NA_OFI_TCP_RXM)
```

Comment:
can we run with TCP without RXM?

Reply:
No, in CaRT we have a list of hardcoded providers: https://github.com/daos-stack/daos/blob/42788347895bdbe9581020e061b3832c7c2ee14f/src/cart/crt_hg.c#L17